test/controllers/api/v2/host_subscriptions_controller_test.rb

```diff
@@ -101,8 +101,11 @@ def test_add_subscriptions
         assert_equal [1], pools_with_quantities[0].quantities.map(&:to_i)
       end
 
-      post :add_subscriptions, params: { :host_id => @host.id, :subscriptions => [{:id => @pool.id, :quantity => "1"}] }
```

Comment:
Did these pass for you locally when running with ktest? I suspect they would fail because the tests won't actually make API calls to Candlepin (unless we record the VCR cassettes). If they are failing, I really think we can omit these test changes.

Reply:
I was working still on it, Cause after adding the subscription for host, total count did not change. Shall I remove it and would not test anything for this change at all?

Reply:
Yeah I'm OK with no unit test for this one. Whosoever verifies the change can do it manually - and the comment you added explains the change so no one will remove that line later on.